runtime/common/src/lib.rs

```diff
@@ -625,6 +633,110 @@ macro_rules! impl_config_traits {
             type XcmExecutor = xcm_executor::XcmExecutor<XcmConfig>;
         }
 
+        // Required for runtime benchmarks
+        pallet_assets::runtime_benchmarks_enabled! {
+            pub struct CustomAssetsBenchmarkHelper;
+
+            impl<AssetIdParameter> pallet_assets::BenchmarkHelper<AssetIdParameter>
+                for CustomAssetsBenchmarkHelper
+            where
+                AssetIdParameter: From<u128>,
+            {
+                fn create_asset_id_parameter(id: u32) -> AssetIdParameter {
+                    (id as u128).into()
+                }
+            }
+        }
+
+        impl pallet_assets::Config<CustomAssetsInstance> for Runtime {
+            type ApprovalDeposit = CustomAssetsApprovalDeposit;
+            type AssetAccountDeposit = CustomAssetsAccountDeposit;
+            type AssetDeposit = CustomAssetsDeposit;
+            type AssetId = AssetId;
+            type AssetIdParameter = Compact<AssetId>;
+            type Balance = Balance;
+            #[cfg(feature = "runtime-benchmarks")]
+            type BenchmarkHelper = CustomAssetsBenchmarkHelper;
+            type CallbackHandle = ();
+            type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
+            type Currency = Balances;
+            type Extra = ();
+            type ForceOrigin = EnsureRootOrTwoThirdsTechnicalCommittee;
+            type Freezer = ();
+            type MetadataDepositBase = CustomAssetsMetadataDepositBase;
+            type MetadataDepositPerByte = CustomAssetsMetadataDepositPerByte;
+            // TODO(#1176): Figure out sensible number after benchmark on reference machine
+            type RemoveItemsLimit = ConstU32<{ 50 }>;
+            type RuntimeEvent = RuntimeEvent;
+            type StringLimit = CustomAssetsStringLimit;
+            type WeightInfo = weights::pallet_assets::WeightInfo<Runtime>;
+        }
+
+        impl pallet_assets::Config<CampaignAssetsInstance> for Runtime {
+            type ApprovalDeposit = CampaignAssetsApprovalDeposit;
+            type AssetAccountDeposit = CampaignAssetsAccountDeposit;
+            type AssetDeposit = CampaignAssetsDeposit;
+            type AssetId = AssetId;
+            type AssetIdParameter = Compact<AssetId>;
+            type Balance = Balance;
+            #[cfg(feature = "runtime-benchmarks")]
+            type BenchmarkHelper = CustomAssetsBenchmarkHelper;
+            type CallbackHandle = ();
+            type CreateOrigin = AsEnsureOriginWithArg<EnsureNever<AccountId>>;
+            type Currency = Balances;
+            type Extra = ();
+            type ForceOrigin = EitherOfDiverse<
+                EnsureRootOrTwoThirdsCouncil,
+                EnsureRootOrAllTechnicalCommittee,
+            >;
```

Comment:
I think we should use the same `ForceOrigin` for all pallets. The distinction between Council and Tech Comm isn't there right now anyways.

Reply:
I disagree for the following reasons:

1. The `CampaignAsset` class is much more powerful, as it allows to spawn assets that can be used to pay transaction fees. Thus it should require at least higher privileges than other asset classes to be created.
2. There are distinctions:
  a. While other asset classes use the `ForceOrigin` solely to technically correct faults, the `CampaignAsset` class uses the `ForceOrigin` to realize business decisions (pushing into directions by utilizing campaigns)
  b. The Council is different from the Technical Committee (TC), even on Zeitgeist. In contrast to the TC, the Council contains all members of the executive. Currently the Council consists of 5 members, with an required agreement from the Council of at least 2/3 to spawn campaign assets, at least two members of the executive have to agree, thus also mapping agreement within the executive of Zeitgeist to move forward with campaigns.

I decided to add the TC to allow assistance in regards to technical aspects, such as cleaning up campaign assets after a campaign is over. However, I think granting the (complete) TC the `ForceOrigin` power invalidates my second argument. I am leaning towards removing the TC completely from the `ForceOrigin` of the `CampaignAsset` class. What's your opinion?

This makes me think about point 1. though. The `MarketAssets` have the `ForceOrigin` `TwoThirdsOrTechnicalCommittee`. Obviously `MarketAssets` have an even more tangible value. `MarketAssets` should never be managed directly as the pallets control the whole lifecycle of market assets such as outcome tokens or pool shares. I added this because I wanted to give the TC the opportunity to hotfix some inconsistent states introduced by pallets. Thinking more about it though, I think the TC should not have the power to manage market assets. What's your opinion?

Makes me also question right now if it makes sense at all to add the TC to any `ForceOrigin`, including `CustomAssets`. After all, with the help of the Council the TC is capable of spawning a referendum with a short period until dispatch.

Reply:
I'm okay with all of these options. As long as there are no elections, I don't see any difference between the Council and TC. In the long run (assuming elections take place), I'd prefer if the TC did not have control over market assets. This is based on the assumption that until then, all bugs will have been fixed or can be fixed by governance rather than TC intervention.

Reply:
Does it make more sense like this: https://github.com/zeitgeistpm/zeitgeist/pull/1177/commits/a86c36af0456b24f37293cb2028cc56126fe65c8

Idea: Still convinced about at least 2/3 Council for CampaignAssets, full tech committee for MarketAssets (due to the potential impact) and 2/3 technical committee for CustomAssets.